packages/esm-patient-allergies-app/src/allergies/allergies-tile.component.tsx

```diff
@@ -10,27 +10,17 @@ interface AllergyTileInterface {
 
 const AllergyTile: React.FC<AllergyTileInterface> = ({ patientUuid }) => {
   const { t } = useTranslation();
-  const { allergies, isLoading } = useAllergies(patientUuid);
+  const { allergies } = useAllergies(patientUuid);
 
-  if (isLoading) {
-    return <InlineLoading role="progressbar" description={`${t('loading', 'Loading')} ...`} />;
-  }
   if (allergies?.length) {
     return (
-      <div>
-        <p className={styles.label}>{t('allergies', 'Allergies')}</p>
-        <p className={styles.content}>
-          <span className={styles.value}>{allergies?.map((allergy) => allergy?.display).join(', ')}</span>
-        </p>
+      <div className={styles.content}>
+        {allergies?.map((allergy) => (
+          <Tile className={styles.tile}>{allergy?.display}</Tile>
```

Comment:
Shouldn't these be [tags](https://react.carbondesignsystem.com/?path=/docs/components-tag--overview) instead?